### ISSUE TITLE
Imagick pecl extension

### ISSUE THREAD

**Issue by oparoz** [kyrofa/owncloud-snap#35]
Some people are against it for security reasons. Ideally it should be in a separate snap, connected via TCP, but I'm not aware of such service/setup.

If we don't include it, people will never be able to add it unless we allow people to build their own extensions (should not be difficult).

So what should we do?

@kyrofa @jospoortvliet @ezraholm50 @enoch85 @miska

**Comment by kyrofa** [comment]
Yeah inter-dependent snaps are a ways out yet (though they're coming). We could always wait for that.

**Comment by oparoz** [comment]
There is also the possibility to just include the module and expose the PHP extension config file. That way users can enable it if they want to.

**Comment by oparoz** [comment]
Requires #40, if we want to include it in this snap
